server/src/test/java/org/elasticsearch/action/admin/indices/TransportAnalyzeActionTests.java

```diff
@@ -250,6 +251,32 @@ public void testFillsAttributes() throws IOException {
         assertEquals("<ALPHANUM>", tokens.get(3).getType());
     }
 
+    public void testMultiTextRequest() throws IOException {
```

Comment:
What is the rationale behind the name of this test? Is there a chance to make it self explanatory about what it tests?

Reply:
In the "request," there should be more than one text and no index name.
```
GET _analyze
{
  "analyzer": "standard", 
  "text": ["a", "b"]
}
```
It could be a better and more descriptive choice. I'll change it to'testAnalyzerWithTwoTextsAndNoIndexName` instead. Thank you, @javanna, for pointing it out.

Reply:
I see, perhaps using terms instead of text here would help :)   the way I see it  text is a set of terms, so it does not make a lot of sense to make it plural.